braitenberg/README.md

```diff
@@ -14,6 +14,19 @@ Your submissions will be sent to the [`lx22-BV1`][challenge] challenge.
 
 [challenge]: https://challenges.duckietown.org/v4/humans/challenges/lx22-BV1
 
+## ℹ️️ Check Robot Compatibility
```

Comment:
I would consider adding this as a subsection to the Testing on a Physical Robot portion of each exercise README to standardize where it appears in the instructions.

Reply:
Can you elaborate a bit more? do you want to name it "Testing on a Physical Robot"? @afdaniele can you weigh in?